.pre-commit-config.yaml

```diff
@@ -16,11 +16,15 @@ repos:
     -   id: pretty-format-json
         args:
         - --autofix
+-   repo: https://github.com/asottile/pyupgrade
+    sha: v1.1.1
```

Comment:
According to the github `v1.2.0` is the latest version. Any reason you're not using that?

Reply:
Nope; I'll go ahead and `pre-commit autoupdate` to bump to latest.